deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -269,7 +269,9 @@ async def create_new_job(
         # TODO could make this if condition `details.version <= "0.25.0"` using the semver package
         if details.version != "0.24.6":
             # max_y param was added in 0.25.0
+            # normalize_y_axis added in 0.25.4
             params.append("max_y")
+            params.append("normalize_y_axis")
```

Comment:
this means it will still be added with versions 0.25.1 and 0.25.2

Reply:
Yeah we really need to start using the semver package to manage all this version conflicts

Reply:
Would adding the _string_version_to_numbers be a good solution for now?

Reply:
The semver package is pretty simple to use, so we should just add it now. You can see an example of using it here https://github.com/CuriBio/pulse3d/blob/2858dbae54772a03ac3543e8456ff6b32c1d4142/src/pulse3D/plate_recording.py#L234